utils/docker/Dockerfile.focal

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:focal
+
+ARG DEBIAN_FRONTEND=noninteractive
+ARG TZ=America/Los_Angeles
+
+# === INSTALL Python ===
+
+RUN apt-get update && \
+    # Install Python
+    apt-get install -y python3 python3-distutils curl && \
+    update-alternatives --install /usr/bin/python python /usr/bin/python3 1 && \
+    curl -sSL https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
```

Comment:
why different lines here?

We currently do the following:
https://github.com/microsoft/playwright/blob/a6fecb5d4b273c934069781c65ae5d50a5007a38/utils/docker/Dockerfile.focal#L16-L20

Reply:
The apt pip is very very old and does not work with our arm afaik.